security/oc_tls.c

```diff
@@ -69,13 +69,50 @@
 #include <mbedtls/debug.h>
 #include <mbedtls/error.h>
 #include <mbedtls/platform.h>
-#endif /* OC_DEBUG */
+#elif OC_DBG_IS_ENABLED || OC_ERR_IS_ENABLED || OC_WRN_IS_ENABLED
+static void
+mbedtls_strerror(int ret, char *buf, size_t buflen)
```

Comment:
I'm surprised SC doesn't complain about includes after a function/variable definition.  I guess we have logs disabled for the analysis build so it doesn't skewer the results, but I would move this to line 85

Reply:
moved